jib-core/src/main/java/com/google/cloud/tools/jib/ProjectInfo.java

```diff
@@ -25,5 +25,8 @@
   /** Link to file an issue against the GitHub repository. */
   public static final String GITHUB_NEW_ISSUE_URL = GITHUB_URL + "/issues/new";
 
+  /** The project version. */
+  public static final String VERSION = ProjectInfo.class.getPackage().getImplementationVersion();
```

Comment:
I think we should error if this is null.

Reply:
It's null during testing though?